styles/_image_compare.scss

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2023 DigitalOcean
+
+Licensed under the Apache License, Version 2.0 (the "License") !default;
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// image_compare embeds
+.image-compare {
+    position: relative;
+}
+
+.image-left,
+.image-right {
+    height: 100%;
+    object-fit: cover;
+    position: absolute;
+    width: 100%;
+}
+
+.image-left {
+    clip-path: polygon(0% 0%, var(--value) 0%, var(--value) 100%, 0% 100%);
+    margin: 0;
+}
+
+.image-right {
+    clip-path: polygon(100% 0%, var(--value) 0%, var(--value) 100%, 100% 100%);
+    margin: 0;
+}
+
+.control-arrow {
+    width: 30px;
+    height: 30px;
+    position: absolute;
+    top: 50%;
+    left: calc(var(--value) - 13px);
+    color: #ffffff;
+    z-index: 90;
+}
+
+.control {
+    background-color: transparent;
+    box-sizing: border-box;
+    font-family: inherit;
+    height: 100%;
+    outline: none;
+    position: absolute;
+    top: 0;
+    width: 100%;
+    z-index: 100;
+}
+
+.control::-moz-range-thumb {
+    background-color: #ffffff;
+    border: none;
+    cursor: ew-resize;
+    height: 100%;
+    width: 2px;
+}
+
+.control::-webkit-slider-thumb {
+    background-color: #ffffff;
+    border: none;
+    cursor: ew-resize;
+    height: 100%;
+    width: 2px;
+}
+
+.control::-moz-range-track {
+    background: transparent;
+    background-size: 100%;
+    box-sizing: border-box;
+}
+
+.control::-webkit-slider-runnable-track {
+    background: transparent;
+    background-size: 100%;
+    box-sizing: border-box;
+    height: 100%;
+}
+
+.control,
+.control::-webkit-slider-runnable-track,
+.control::-webkit-slider-thumb {
+    appearance: none;
+}
```

Comment:
These should all be scoped within the `.image-compare` class, please

Reply:
Moved the declarations inside the container class.